packages/mobile/src/invite/saga.test.ts

```diff
@@ -196,3 +199,25 @@ describe(watchRedeemInvite, () => {
       .run()
   })
 })
+
+describe(generateLink, () => {
+  beforeEach(() => {
```

Comment:
Nit: why do you need this since you only have one test?

Reply:
How about we enable clearing mocks between tests using the config? 
https://jestjs.io/docs/en/configuration#clearmocks-boolean

Reply:
It has to be there because otherwise the mock calls number is set on the other tests that run before.  I like the `beforeEach` because if you add more tests you don't have to copy it over.